frontend/router/index.js

```diff
@@ -0,0 +1,23 @@
+import { createRouter, createWebHashHistory } from "vue-router";
+
+const routes = [
+  {
+    path: "/",
+    name: "Home",
+    component: () => import(/* webpackChunkName: "home" */ "@/views/Home.vue"),
```

Comment:
Remove comments.

Reply:
I would stay with these comments here. Thanks to thtm, webpack will know how to name chunk files, eg: https://github.com/krzysztofrewak/browary/tree/public/js

Reply:
These are magic comments used to name chunks.